### PR TITLE
WCF initial integration

### PR DESCRIPTION
**Pull request by boydc7**
Initial WCF - I removed all the cruft from the WCF extensibility services and other integration points that were attempted (in another branch) and left just what should be working for what has been discussed in slack/etc.  I was able to actually get a collection of HTTP headers after some deeper exploration, so I believe distributed tracing should be working there as well (ONLY in an HTTP transport naturally, which applies to much of the info (tags, res name, etc.)). 

Some questions to decide on:

span.Type - currently it's set to Web for both HTTP transport and all others.  Should we add more, use different, etc.?

span.ResourceName - in an HTTP transport, currently following as close to a "HTTPVerb Path/Action" (similar to other http requests).  For others, I'm currently setting it to the entire request message (akin to a db query I suppose).  Should this be adapted anywhere?

### CODE REVIEW

File: src/Datadog.Trace.ClrProfiler.Managed/Models/WcfHttpRequestMessageSpanIntegrationDelegate.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            {
+                return;
+            }
+
+            DependencyFactory.SpanDecorationService(this).Decorate(Scope.Span, this);
+        }
+
+        public override void OnEnd()
+        {
+            var x = _httpRequestMessageProperty.Headers;
```
Comment (lucaspimentel) [inline src/Datadog.Trace.ClrProfiler.Managed/Models/WcfHttpRequestMessageSpanIntegrationDelegate.cs]: If this line is intentional, can you leave a comment explaining why it's needed so it's not removed accidentally?

File: integrations.json [hunk 2]
```diff
@@ -452,5 +452,24 @@
...
+    "method_replacements": [
+      {
+        "caller": {},
+        "target": {
+          "assembly": "System.ServiceModel",
+          "type": "System.ServiceModel.Dispatcher.ChannelHandler",
+          "method": "HandleRequest"
+        },
+        "wrapper": {
+          "assembly": "Datadog.Trace.ClrProfiler.Managed, Version=0.8.2.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
```
Comment (lucaspimentel) [inline integrations.json]: We're up to `Version=1.0.0.0` now.

File: src/Datadog.Trace.ClrProfiler.Managed/Models/WcfHttpRequestMessageSpanIntegrationDelegate.cs (hunk 1 above)
Comment (boydc7) [reply src/Datadog.Trace.ClrProfiler.Managed/Models/WcfHttpRequestMessageSpanIntegrationDelegate.cs]: Ha, nope, must have been a debug leftover...it's gone

File: integrations.json (hunk 2 above)
Comment (boydc7) [reply integrations.json]: Updated and merged develop

**Pull request merged**